client/ayon_maya/api/lib.py

```diff
@@ -4247,7 +4246,7 @@ def _get_parent(reference_node):
     return parents
 
 
-def get_attr(node, attr, default=None):
+def get_instance_node_attr(node, attr, default=None):
```

Comment:
I understand you're trying to avoid the conflict with `get_attribute` but in reality this code has nothing to do with an `instance node`. @antirotor can you guide along perhaps - I'm too short on time to check what's a decent name and code structure here - and I feel with just a quick "heads up" from my end in the reviews it's just costing Kayla more time than needed.